examples/platform/silabs/efr32/rs911x/sl_wifi_if.c

```diff
@@ -0,0 +1,754 @@
+    return SL_STATUS_OK;
+}
+
+#if CHIP_DEVICE_CONFIG_ENABLE_SED
+/******************************************************************
+ * @fn   wfx_rsi_power_save()
+ * @brief
+ *       Setting the RS911x in DTIM sleep based mode
+ *
+ * @param[in] None
+ * @return
+ *        None
+ *********************************************************************/
+int32_t wfx_rsi_power_save()
+{
+    int32_t status;
+    status = rsi_bt_power_save_profile(2, 0);
+    if (status != RSI_SUCCESS)
+    {
+        SILABS_LOG("BT Powersave Config Failed, Error Code : 0x%lX", status);
+        return status;
+    }
+
+    sl_wifi_performance_profile_t wifi_profile = { ASSOCIATED_POWER_SAVE };
+    status = sl_wifi_set_performance_profile(&wifi_profile);
+    if (status != RSI_SUCCESS)
+    {
+        SILABS_LOG("Powersave Config Failed, Error Code : 0x%lX", status);
+        return status;
+    }
+    SILABS_LOG("Powersave Config Success");
+    return status;
+}
+#endif /* CHIP_DEVICE_CONFIG_ENABLE_SED */
+
+/*************************************************************************************
+ * @fn  static int32_t wfx_wifi_rsi_init(void)
+ * @brief
+ *      Wifi initialization called from app main
+ * @param[in]  None
+ * @return
+ *        None
+ *****************************************************************************************/
+int32_t wfx_wifi_rsi_init(void)
+{
+  SILABS_LOG("wfx_wifi_rsi_init started");
+  sl_status_t status;
+  status = sl_wifi_init(&config  , default_wifi_event_handler);
+  if(status != SL_STATUS_OK){
+    SILABS_LOG("wfx_wifi_rsi_init failed %x", status);
+    return status;
+  }
+  return status;
+}
+
+/*************************************************************************************
+ * @fn  static int32_t wfx_rsi_init(void)
+ * @brief
+ *      driver initialization
+ * @param[in]  None
+ * @return
+ *        None
+ *****************************************************************************************/
+static int32_t wfx_rsi_init(void)
+{
+    sl_status_t status;
+    status = wfx_wifi_rsi_init();
+    if(status != SL_STATUS_OK) {
+      SILABS_LOG("wfx_rsi_init failed %x", status);
+      return status;
+    }
+
+    status = sl_wifi_get_mac_address(SL_WIFI_CLIENT_INTERFACE, (sl_mac_address_t *) &wfx_rsi.sta_mac.octet[0]);
+    if (status != SL_STATUS_OK)
+    {
+        SILABS_LOG("sl_wifi_get_mac_address failed: %x", status);
+        return status;
+    }
+
+    wfx_rsi.events = xEventGroupCreateStatic(&rsiDriverEventGroup);
+    wfx_rsi.dev_state |= WFX_RSI_ST_DEV_READY;
+    osSemaphoreRelease(sl_rs_ble_init_sem);
+    return status;
+}
+
+/*************************************************************************************
+ * @fn  void wfx_show_err(char *msg)
+ * @brief
+ *      driver shows error message
+ * @param[in]  msg
+ * @return
+ *        None
+ *****************************************************************************************/
+void wfx_show_err(char * msg)
```

Comment:
this function is not used anywhere.
We can remove this and directly add SILABS_LOG